common/lib/xmodule/xmodule/tests/__init__.py

```diff
@@ -251,30 +259,103 @@ def setUp(self, *args, **kwargs):
 
```

Comment:
`_manager` seems like a dangerous name for a deeply nested and mixed-in hierarchy.  This might be the first time I recommend using `__manager` as a name.